pkgs/applications/video/mpv/scripts/mpv-notify-send.nix

```diff
@@ -13,9 +13,9 @@ buildLua rec {
 
   src = fetchFromGitHub {
     owner = "emilazy";
-    repo = pname;
+    repo = "mpv-notify-send";
     rev = "a2bab8b2fd8e8d14faa875b5cc3a73f1276cd88a";
-    sha256 = "sha256-EwVkhyB87TJ3i9xJmmZMSTMUKvfbImI1S+y1vgRWbDk=";
+    hash = "sha256-EwVkhyB87TJ3i9xJmmZMSTMUKvfbImI1S+y1vgRWbDk=";
   };
 
   patches = [
```

Comment:
We could also ditch the patch since it has been [merged](https://github.com/Parranoh/mpv-notify-send/commit/d98d9fe566b222c5b909e3905e9e201eaec34959)

Reply:
That's a different repo

Reply:
Oh nvm the repo ownership was transferred. Thanks for catching.